openwisp_notifications/static/openwisp-notifications/js/unsubscribe.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
```

Comment:
I don't prefer the added complexity of JS when we can achieve the same functionality by using a simple HTML form. It would require reloading of the page on form submission, but I don't see a problem with that.

Reply:
As discussed in our call, I have added selenium tests to test the API requests.